extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -230,68 +287,181 @@ describe('amp-a4a', () => {
       }).onFirstCall().returns(Promise.resolve(mockResponse));
       return createAdTestingIframePromise().then(fixture => {
         const doc = fixture.doc;
-        const a4aElement = createA4aElement(doc);
+        a4aElement = createA4aElement(doc);
         a4aElement.setAttribute('width', 200);
```

Comment:
Done.  Also got rid of a number of `body.appendChild` because it's done in `createA4AElement`.